### ISSUE TITLE
Client/Server mode: Mapping for font resizing

### ISSUE THREAD

**Issue by phi-mah** [neovide/neovide#856]
Hi,
and sorry for the zillionth font question...

I make heavy use of the `--remote-tcp` option, connecting to a headless nvim session on a remote sever. 

To quickly  change the visible font size, i have a function as follows, which is eventually mapped to `<C-+>/<C-->` . 
```
let s:guifontsize = 10
let s:guifont = "JetBrainsMono\\ Nerd\\ Font"

function! AdjustFontSize(amount)
  let s:guifontsize = s:guifontsize+a:amount
  exe "set guifont=" .. s:guifont .. ":h" .. s:guifontsize
endfunction
```

This approach works in a *native* neovide session, but not when connecting to a *remote* nvim instance. Still, I can change the font size via the direct colon statement, e.g.:`:set guifont="JetBrainsMono\\ Nerd\\ Font:h16"`, but this is a bit cumbersome for a statement that I use quite frequently.

Coming from `neovim-qt`, a similar function as given above did work. There the [`neovim-gui-shim`](https://github.com/equalsraf/neovim-gui-shim) plugin sends a rpc to all connected clients. I assume that `neovide` handles this differently which prevents the function statements from reaching the GUI.

In short: It would be nice to have a shortcut that changes the font size and works even with a TCP connection in between.

**Comment by Kethku** [comment]
This is almost certainly an upstream bug. I see no reason why the given function should operate differently local vs remote.

**Comment by Kethku** [comment]
@phi-mah does this issue still happen? Its possible this was a threading issue, but I'd like to follow up. Please reopen if so
